.github/CODEOWNERS

```diff
@@ -321,6 +320,9 @@
 /comp/snmpscan @DataDog/ndm-core
 # END COMPONENTS
 
+# Additional notification to  @iglendd about Agent Telemetry changes for optional approval and governance acknowledgement
+/comp/core/agenttelemetry               @iglendd
```

Comment:
@chouetz will 2nd reference to /comp/core/agenttelemetry notify both ASC and myself about the chang (I cannot test it myself). As part of component ownership generation ASC is the owner of the /comp/core.

Reply:
If you add that, only you will be notified. More precise rules overwrite less precise ones. If you want both to be notified you need to change your line to
```
/comp/core/agenttelemetry               @iglendd @DataDog/agent-shared-components
```